torch/csrc/jit/script/compiler.cpp

```diff
@@ -727,6 +728,30 @@ static Value* emitBuiltinNode(
   return packOutputs(graph, n->outputs());
 }
 
+using OperatorMap = std::pair<Symbol, std::vector<std::shared_ptr<Operator>>>;
+
+std::multimap<unsigned, OperatorMap> findOperatorsWithSimilarName(Symbol name) {
+  std::string user_function_name(name.toQualString());
+
+  // currently only using the name of the op, could be improved to check
+  // similarity of arguments
+  using ranking_pair = std::pair<unsigned, OperatorMap>;
+  std::multimap<unsigned, OperatorMap> rankings;
+
+  auto all_operators = getAllOperators();
+  const size_t MAX_EDIT_DIST = 2u;
+  auto best_dist = MAX_EDIT_DIST;
+  for (auto op: all_operators) {
+    std::string builtin_op_name = op.first.toQualString();
+    auto edit_dist = computeEditDistance(user_function_name, builtin_op_name, MAX_EDIT_DIST);
+    if (edit_dist <= MAX_EDIT_DIST) {
```

Comment:
How does this work? Looks like we're never updating `best_dist`. So we're just adding everything under the MAX.